airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -226,13 +235,15 @@ private CancellationScope generateSyncWorkflowRunnable(final ConnectionUpdaterIn
         }
 
         prepareForNextRunAndContinueAsNew(connectionUpdaterInput);
+        metricClient.count(OssMetricsRegistry.TEMPORAL_WORKFLOW_RESTART_SUCCESS, 1L);
       } catch (final ChildWorkflowFailure childWorkflowFailure) {
         // when we cancel a method, we call the cancel method of the cancellation scope. This will throw an
         // exception since we expect it, we just
         // silently ignore it.
         if (childWorkflowFailure.getCause() instanceof CanceledFailure) {
           // do nothing, cancellation handled by cancellationScope
-
+          metricClient.count(OssMetricsRegistry.TEMPORAL_WORKFLOW_RESTART_FAILURE, 1L,
+              new MetricAttribute(RESET_WORKFLOW_FAILURE_ATTRIBUTE_KEY, "CANCELED"));
```

Comment:
nit: I find the `_ATTRIBUTE_KEY` not super clear, what do you think of `RESET_WORKFLOW_FAILURE_CAUSE`?

Reply:
Per @davinchia 's comment above, this is going to move to the `MetricTags` class and match the naming conventions used there.